### DISCUSSION TITLE
pdf form slides?

### DISCUSSION THREAD

**Discussion by Reben80** [TeamBasedInquiryLearning/linear-algebra#228] [Ideas]
I used linear algebra for the two semesters and one things I like is able to download the slides as pdf. One reason for this is to be able to write on it while presenting in class.

**Comment by StevenClontz** [comment]
Sorry for slow response!

Right now the slideshow output is via PreTeXt, which only has real support for Reveal.js HTML slidedecks.

I typically prefer taking screengrabs from the materials and putting them in a Jamboard where I have a lot of flexibility in annotating things, responding to students, shuffling things around. Beamer and Reveal.js are both not very flexible. But I know others like them.

**Comment by siwelwerd** [comment]
Circling back to add that this has been implemented and will be available in the 2024 spring preview edition.  See [here](https://github.com/TeamBasedInquiryLearning/linear-algebra/blob/gh-pages/2024/pdf/TBIL-LA-2024-slides.pdf) for a preview.
